aspnetcore-authentication-jwtbearer/src/Directory.Build.props

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+
+  <Import Project="../../src.props" />
+
+  <PropertyGroup>
+    <PackageTags>OAuth 2.0;OpenID Connect;Security;ASP.NET Core;DPoP</PackageTags>
+    <Product>Duende BFF</Product>
```

Comment:
Probably a copy paste issue? Should probably be: Duende.AspNetCore.Authentication.JwtBearer

Reply:
Good spot